src/components/v2/shared/FundingCycleConfigurationDrawers/NFTDrawer/NFTUpload.tsx

```diff
@@ -0,0 +1,123 @@
+import { t, Trans } from '@lingui/macro'
+import { Form, FormInstance, Upload } from 'antd'
+import { RcFile } from 'antd/lib/upload'
+import TooltipLabel from 'components/shared/TooltipLabel'
+import { CSSProperties, useContext, useState } from 'react'
+
+import { LoadingOutlined, UploadOutlined } from '@ant-design/icons'
+import { ThemeContext } from 'contexts/themeContext'
+import { emitErrorNotification } from 'utils/notifications'
+import { ipfsCidUrl, pinFileToIpfs } from 'utils/ipfs'
+
+const ALLOWED_FILE_TYPES = [
+  'image/jpeg',
+  'image/png',
+  'image/gif',
+  'image/svg',
```

Comment:
**issue** Uploading SVG is a potential vulnerability.

I'd suggest we might not support SVG here and in the cloud function for now.

cc @tankbottoms 

http://ghostlulz.com/xss-svg/

Reply:
Removed for now